recipes/docker_registry.rb

```diff
@@ -156,6 +156,57 @@
   only_if { File.exist? "#{Chef::Config['file_cache_path']}/#{base_filename}" }
 end
 
+#git
+git_image_url = node['hops']['docker']['git']['download_url']
+git_filename = File.basename(git_image_url)
+download_git_command = " wget #{git_image_url}"
+git_image = "#{node['hops']['docker']['git']['image']['name']}:#{node['hops']['docker_img_version']}"
+
+if node['install']['enterprise']['install'].casecmp? "true"
+  git_image_url ="#{node['install']['enterprise']['download_url']}/docker-tars/#{node['hops']['docker_img_version']}/#{git_filename}"
+  download_git_command = " wget --user #{node['install']['enterprise']['username']} --password #{node['install']['enterprise']['password']} #{git_image_url}"
+end
+
+
+bash "download_git_image" do
+  user "root"
+  sensitive true
+  code <<-EOF
+       #{download_git_command} -O #{Chef::Config['file_cache_path']}/#{git_filename}
```

Comment:
Again, overwrite it if it's already there. In case there was a problem downloading it (incomplete file) - easier to just delete and download again.